build_tools/cmake/iree_macros.cmake

```diff
@@ -123,3 +123,51 @@ function(iree_select_compiler_opts OPTS)
   endif()
   set(${OPTS} ${_OPTS} PARENT_SCOPE)
 endfunction()
+
+#-------------------------------------------------------------------------------
+# Data dependencies
+#-------------------------------------------------------------------------------
+
+# Adds 'data' dependencies to a target.
+#
+# Parameters:
+# NAME: name of the target to add data dependencies to
+# DATA: List of targets and/or files in the source tree. Files should use the
+#       same format as targets (i.e. iree::package::subpackage::file.txt)
+function(iree_data)
```

Comment:
Can we call this something that makes its role a little clearer? add_data_dependency?

Reply:
Renamed to `iree_add_data_dependencies()`